R/importance.R

```diff
@@ -36,131 +39,243 @@
 #'  the risk with the permuted/removed X divided by observed risk with all X; or
 #'  \code{"difference"}, which returns the difference between the risk with the
 #'  permuted/removed X and the observed risk.
+#' @param covariate_groups Optional named list covariate groups which will
+#'  invoke variable importance evaluation at the group-level, by
+#'  removing/permuting all covariates in the same group together. If covariates
+#'  in the task are not specified in the list of groups, then those covariates
+#'  will be added as additional single-covariate groups.
+#' @param B Optional integer specifying the number of permutations to average
+#'  the permuted risk over when \code{type = "permute"}.
+#' @param cores Number of cores for parallelizing the \code{B} permutations.
```

Comment:
remove this argument. consider replacing with a note to set the parallelization scheme via a `future::plan()` specification.